string detector_ns
Frame[] frames